src/main/java/org/breedinginsight/model/User.java

```diff
@@ -63,6 +69,7 @@ public static User parseSQLRecord(Record record, @NotNull BiUserTable tableName)
                 .name(record.getValue(tableName.NAME))
                 .email(record.getValue(tableName.EMAIL))
                 .systemRoles(new ArrayList<>())
+                .programRoles(new ArrayList<>())
                 .active(record.getValue(tableName.ACTIVE))
```

Comment:
There is a mismatch with the active field between here and the api docs. The response here includes it but the api docs schema and example doesn't.

Reply:
changed and pushed